docs/src/Parser/Parser.php

```diff
@@ -87,7 +87,6 @@ public function parse_files( $files, $root ) {
 			$out = [
 				'file' => $this->export_docblock( $file ),
 				'path' => str_replace( DIRECTORY_SEPARATOR, '/', $file->getFilename() ),
-				'root' => $root,
```

Comment:
I'm not sure if this key is used anywhere.

With this change the the `root` key for each `file` object in the generated JSON file would no longer exist, preventing any unnecessary change to the JSON if it were generated in a different environment from where the docs command was previously run.